README.md

```diff
@@ -34,45 +30,52 @@ Add this to your package's pubspec.yaml file:
 dependencies:
   card_scanner: <latest-version>
 ```
-> get the [latest version number here](https://pub.dev/packages/card_scanner#-installing-tab-)
 
+> get the [latest version number here](https://pub.dev/packages/card_scanner#-installing-tab-)
 
 ## Usage
-Just import the package and call `scanCard`
+
+Just import the package and call `scanCard`:
 
 ```dart
 import 'package:card_scanner/card_scanner.dart';
 var cardDetails = await CardScanner.scanCard()
 
 print(cardDetails)
 ```
-Example Output : 
+
+Example Output:
+
 ```dart
-Card Number = 5173949117389006 
+Card Number = 5173949117389006
 Expiry Date = 11/26
 ```
 
-The above code opens the device camera , looks for a valid card and gets the required details and returns the `CardDetails` object
+The above code opens the device camera, looks for a valid card and gets the required details and returns the `CardDetails` object.
 
 ---
 
-### Scan Options : 
-If you wish to obtain the card holder name and card issuer , you can specify the options.
+### Scan Options
+
+If you wish to obtain the card holder name and card issuer, you can specify the options:
+
 ```dart
 import 'package:card_scanner/card_scanner.dart';
 var cardDetails = await CardScanner.scanCard(
-    scanOptions: CardScanOption (
-        scanCardHolderName: true, 
-        scanCardIssuer: true
-    )
+    scanOptions: CardScanOptions(
+        scanCardHolderName: true,
+        scanCardIssuer: true,
+    ),
```

Comment:
info: the main purpose was this update, just to help out potential consumers, the rest was mostly added by auto-formatter.